code/modules/mob/living/simple_animal/friendly/fox.dm

```diff
@@ -6,23 +6,23 @@
 	icon_state = "fox"
 	icon_living = "fox"
 	icon_dead = "fox_dead"
-	speak = list("Ack-Ack","Ack-Ack-Ack-Ackawoooo","Geckers","Awoo","Tchoff")
-	speak_emote = list("geckers", "barks")
-	emote_hear = list("howls.","barks.")
+	speak = list("AAAAAAAAAAAAAAAAAAAA","Yip","Hehehehehe")
+	speak_emote = list("screamss")
```

Comment:
screams

Reply:
not exactly, at least not ones that like...make sense in a sentence